postgres/datadog_checks/postgres/statements.py

```diff
@@ -181,6 +194,35 @@ def _get_pg_stat_statements_columns(self):
                 self._stat_column_cache = col_names
                 return col_names
 
+    def _check_called_queries(self):
+        pgss_view_without_query_text = self._config.pg_stat_statements_view
+        if pgss_view_without_query_text == "pg_stat_statements":
+            pgss_view_without_query_text = "pg_stat_statements(false)"
```

Comment:
Just for future reference, can we add a comment here to explain why we need to use `pg_stat_statements(false)`?

Reply:
Definitely.